event-sourcing/event-publisher/linked-event-processor/src/main/java/uk/gov/justice/services/eventsourcing/linkedevent/ActiveStreamsProcessor.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.services.eventsourcing.linkedevent;
+
+import uk.gov.justice.services.eventsourcing.repository.jdbc.event.EventJdbcRepositoryFactory;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStream;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamJdbcRepository;
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamJdbcRepositoryFactory;
+import uk.gov.justice.subscription.registry.DefaultEventSourceDefinitionFactory;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import javax.inject.Inject;
+
+public class ActiveStreamsProcessor {
+
+    @Inject
+    private LinkedEventsProcessor linkedEventsProcessor;
```

Comment:
We normally do @Inject as package protected...see all Injections in this class too

Reply:
fixed